pages/borrow/index.tsx

```diff
@@ -302,74 +297,44 @@ const Markets: NextPage = () => {
                 honeyClient,
                 honeyMarket,
                 honeyUser,
-                parsedReserves
+                parsedReserves,
+                mData
               );
 
               collection.openPositions = await handlePositions(
                 collection.verifiedCreator,
                 userOpenPositions
               );
 
-              collection.rate =
-                (await getInterestRate(
-                  collection.utilizationRate,
-                  collection.id
-                )) || 0;
-
               if (currentMarketId === collection.id) {
+                setActiveInterestRate(collection.rate);
                 // @ts-ignore
-                setNftPrice(RoundHalfDown(collection.nftPrice));
+                setNftPrice(RoundHalfDown(Number(collection.nftPrice)));
```

Comment:
still BN or string? is conversion to Number required?

Reply:
fixed, bn was not needed, nor number